tools/js-tools/eslintrc/react.js

```diff
@@ -23,10 +23,6 @@ module.exports = {
 		'react/prefer-es6-class': 'warn',
 
 		// Temporarily override plugin:@wordpress/react so we can clean up failing stuff in separate PRs.
-		'react-hooks/exhaustive-deps': [ 'warn', { additionalHooks: '' } ],
 		'react/jsx-key': 'off',
-		'react/no-direct-mutation-state': 'off',
-		'react/no-find-dom-node': 'off',
```

Comment:
Did I miss something or was this never causing issues to begin with?

Reply:
Ah, you mention that in the PR body. Helps to read the PR before jumping into the code, I guess...

Reply:
Back in #38436 there was one thing left triggering it, but that got fixed by #38339.